<launch>

  <!-- Start the simulator node. This publishes the sensor data. -->
  <node name="range_sensor_simulator" pkg="fuse_tutorials" type="range_sensor_simulator" output="screen"/>
  
  <!-- Start rviz -->
  <node name="rviz" pkg="rviz" type="rviz" output="screen"
        args="-d $(find fuse_tutorials)/config/range_sensor_tutorial.rviz"/>

  <!-- Start fuse fixed-lag smoother -->
  <node name="state_estimation" pkg="fuse_optimizers" type="step_wise_fixed_lag_smoother_node" output="screen">
    <rosparam command="load" file="$(find fuse_tutorials)/config/range_sensor_stepwise_optimization_tutorial.yaml"/>
  </node>

</launch>
